src/process/win.cc

```diff
@@ -7,6 +7,27 @@
 
 namespace SSC {
 
+SSC::String FormatError(DWORD error, SSC::String source) {
```

Comment:
nice, much needed

Reply:
Agree, we want to move this to a more central location though, eventually.